configs/addons/counterstrikesharp/gamedata/gamedata.json

```diff
@@ -133,6 +133,12 @@
       "linux": "55 48 89 E5 41 57 41 56 41 55 41 54 49 89 FC 53 48 81 EC 88 00 00 00 48 8D 05 ? ? ? ?"
     }
   },
+  "CCSGameRules_FindPickerEntity": {
+    "offsets": {
+      "windows": 28,
```

Comment:
Unable to confirm this myself, but windows offset should be `27` according to [CS2Fixes' gamedata](https://github.com/Source2ZE/CS2Fixes/blob/e51d515719dfae562ff9595497fc44caff6bbfa2/gamedata/cs2fixes.games.txt#L403-L407)

Reply:
Corrected it, thanks